GameData/RP-1/Programs/Programs.cfg

```diff
@@ -928,10 +930,10 @@ RP0_PROGRAM
 	}
 }
 
-disabled_RP0_PROGRAM
+RP0_PROGRAM
 {
 	name = JupiterMoonLandings
-	title = Jovian Moon Landings (No Content Yet)
+	title = Jovian Moon Landings (WIP)
```

Comment:
I don't think we should be enabling programs with TODO descriptions? :D
(Also I thought Max gave feedback on the contracts, not the program length & funding etc?)

Reply:
We already have some with TODO descriptions enabled, IIRC, but I can redisable them in a bit (or you can do it yourself if you want to release soon)

Reply:
Oh jeez we do? I thought that was part of your mass disabling pass before initial release. Whoops! And no, not in a hurry, I need to do the unlock credit curve stuff and fix the bug siimav reported with credit.